src/pages/integrations/invoice/confirm-invoice/confirm-invoice.ts

```diff
@@ -217,14 +202,6 @@ export class ConfirmInvoicePage extends ConfirmPage {
     this.initialize(wallet).catch(() => {});
   }
 
-  ionViewWillLeave() {
-    if (!this.invoicePaid) {
-      this.clipboardProvider.copy(this.invoiceUrl);
-    } else {
-      this.externalLinkProvider.open(this.browserUrl);
-    }
-  }
-
```

Comment:
This fixes the `https://test.bitpay.com/invoice/jasdfasdf9879asdf` copied in `home.ts` page instead of `https://test.bitpay.com/i/jasdfasdf9879asdf`